db-client-java/src/main/java/com/eventstore/dbclient/PersistentSubscriptionListener.java

```diff
@@ -1,12 +1,15 @@
 package com.eventstore.dbclient;
 
 public abstract class PersistentSubscriptionListener {
-    public void onEvent(PersistentSubscription subscription, ResolvedEvent event) {
+    public void onEvent(PersistentSubscription subscription, int retryCount, Position position, ResolvedEvent event) {
     }
 
     public void onError(PersistentSubscription subscription, Throwable throwable) {
     }
 
     public void onCancelled(PersistentSubscription subscription) {
     }
+
+    public void onConfirmation(PersistentSubscription subscription) {
```

Comment:
**QUESTION:** Is this callback invoked after ACK?

Reply:
Nope, it's called when the server confirms the subscription has started. There is a specific gRPC message for that.

Reply:
Maybe we could name it `onStarted`? Do you know if/how other clients are naming it?

Reply:
The only client that I can find that exposes this is the NodeJS one, which uses `confirmation`, but the other clients use `confirmation` internally. I'd be inclined to keep this named the same as the message coming over gRPC for now